test/e2e/framework.go

```diff
@@ -33,7 +33,6 @@ import (
 	"github.com/containernetworking/plugins/pkg/ip"
 	log "github.com/sirupsen/logrus"
 	"golang.org/x/mod/semver"
-	"gopkg.in/yaml.v2"
```

Comment:
removed by accident? Golangci-lint is failed by this deletion.

Reply:
fixed